### DISCUSSION TITLE
How to disable default wrapping in markdown files

### DISCUSSION THREAD

**Discussion by LintaoAmons** [LazyVim/LazyVim#1057] [Q&A] [answered]
I put `vim.opt.wrap = false` to options.lua but not working?
How can I set nowrap in lazyvim~

**Comment by dpetka2001** [comment]
What do you mean? LazyVim has `wrap = false` set by default. No need for you to set it. See [options](https://www.lazyvim.org/configuration/general#options).

**Reply by LintaoAmons** [reply]
Oh, then must be some plugin changed the settings. Let me find out

**Reply by LintaoAmons** [reply]
OK I found that only markdown have this `warp` setting?

**Reply by folke** [reply, marked as answer]
Add this to your autocmds.lua:

`vim.api.nvim_create_augroup("lazyvim_wrap_spell", {clear = true})`
